packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -49,18 +47,18 @@ BuildRequires: rubygem(apipie-rails) >= 0.5.14
 BuildRequires: rubygem(fx) < 1.0
 BuildRequires: rubygem(pg)
 BuildRequires: rubygem(anemone)
-BuildRequires: rubygem(pulpcore_client) >= 3.28.0
-BuildRequires: rubygem(pulpcore_client) < 3.29.0
-BuildRequires: rubygem(pulp_file_client) >= 1.14.0
-BuildRequires: rubygem(pulp_file_client) < 1.15.0
-BuildRequires: rubygem(pulp_ansible_client) >= 0.18.0
-BuildRequires: rubygem(pulp_ansible_client) < 0.19.0
-BuildRequires: rubygem(pulp_container_client) >= 2.15.0
-BuildRequires: rubygem(pulp_container_client) < 2.16.0
+BuildRequires: rubygem(pulpcore_client) >= 3.39.0
+BuildRequires: rubygem(pulpcore_client) < 3.40.0
```

Comment:
no `with` here?

Reply:
ah, this is because these come from gem2rpm and that didn't work you said, kk

Reply:
exactly - I'm trying to figure out how to do this as cleanly as possible (and later submit an update to gem2rpm).